keyboards/eyeohdesigns/babyv/readme.md

```diff
@@ -0,0 +1,19 @@
+# babyV
+
+![babyV](https://i.imgur.com/iRUm9I5l.jpg)
+
+babyV is a 40% ortholinear take on the EM7/Alice layout.
+
+* Keyboard Maintainer: [Eye Oh Designs](https://github.com/joedinkle)
+* Hardware Supported: babyV ver 1.00
+* Hardware Availability: Groupbuy ran from Nov - Dec 2020
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader to flash new firmware onto the board?

Reply:
Do you mean the location of a reset button or keycode?

Reply:
Yup, or if bootmagic is enabled, and how to trigger it.

Reply:
Requested changes made.